config/monitoring/prometheus/apps/prometheus-configs.yaml

```diff
@@ -1240,6 +1259,20 @@ data:
             triage: 'https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Distributed-Workloads/kuberay-operator-availability.md'
             summary: Alerting for KubeRay Operator
 
+  kueue-alerting.rules: |
+    groups:
+    - name: Distributed Workloads Kueue
+      interval: 1m
+      rules:
+        - alert: Kueue Operator is not running
+          expr: absent(up{job=~'Kueue Operator'}) or up{job=~'Kueue Operator'} != 1
+          labels:
+            severity: warning
+          annotations:
+            description: This alert fires when the Kueue Operator is not running.
+            triage: 'https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Distributed-Workloads/kuberay-operator-availability.md'  # TODO: what link here?
```

Comment:
i think you will need to create a new file for kueue in **https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Distributed-Workloads** and link it here.

Reply:
as long as this is updated, i am fine with this PR